cmd/terramate/e2etests/cloud/run_cloud_deployment_test.go

```diff
@@ -345,13 +373,17 @@ func TestCLIRunWithCloudSyncDeployment(t *testing.T) {
 				genIdsLayout = tc.layout
 			}
 
```

Comment:
```suggestion
			// needed for invoking `terraform ...` commands in the sandbox tree (see tc.layout)
```

It wasn't obvious to me at first, maybe makes sense to add a note why this is needed.
But maybe it's just me, feel free to ignore.